packages/node_modules/@ciscospark/redux-module-media/src/reducer.js

```diff
@@ -55,7 +74,15 @@ export default function reducer(state = initialState, action) {
       .setIn([`remoteStream`, `url`], call.remoteMediaStreamUrl);
   }
   case CALL_DISCONNECTED: {
-    return initialState;
+    const isListening = state.getIn([`status`, `isListening`]);
+    const incomingAcknowledged = state.getIn([`incomingCall`, `hasAcknowledged`]);
+    const incomingStatus = state.getIn([`status`, `isRinging`]);
+    const incomingCall = state.getIn([`incomingCall`, `call`]);
+    return initialState
```

Comment:
Just in case a call was ringing us when the disconnect happens, don't get rid of it.